views/edit.php

```diff
@@ -0,0 +1,47 @@
+<?php
+require_once('../controllers/variable.php');
+require_once('../controllers/db_connect.php');
+require_once('../models/todomodel.php');
+
+$db_connect = new DbConnect();
+$dbh = $db_connect->getDbConnection();
+
+$todoModel = new TodoModel();
+
+
+$variable = new Variable();
+$id = $variable->createGet('id');
+
+$todoModel->get($id);
+$data = $todoModel->get($_GET['id']);
```

Comment:
ここ、同じことやってますかね？
```suggestion
$data = $todoModel->get($get);
```

これだけでいいような気がします。